<launch>
  <arg name="output" default="log"/>
  <arg name="state_estimation" default="false"/>
  <arg name="standalone" default="false"/>

  <!-- World setup -->
  <include if="$(arg standalone)" file="$(find agimus_demos)/launch/tiago_deburr_world_setup.launch">
    <arg name="simulation" value="false"/>
  </include>

  <!-- Setup vision -->
   <group ns="vision">
    <node pkg="agimus_vision" type="tracker_object"
        output="$(arg output)"
        name="tracker"
        >
        <param name="imageTopic"      value="/xtion/rgb/image_raw" />
        <param name="cameraInfoTopic" value="/xtion/rgb/camera_info" />
        <param name="debugDisplay"    value="true" />
        <param name="broadcastTopic"  value="true" />
        <param name="broadcastTf"     value="true" />
        <param name="broadcastTfPostfix" value="_tf" />
        <param name="objectType" value="apriltag" />
        <param name="cameraFrame" value="xtion_optical_frame" />
    </node>


  </group>
</launch>
